frontend/src/components/bucket/BucketTable.vue

```diff
@@ -38,8 +38,8 @@ const showPermissions = async (bucketId: string, bucketName: string) => {
   permissionBucketName.value = bucketName;
 };
 
-const displayPermissionsIcon = (bucket: Bucket) => {
-  return permissionStore.getIsBucketActionAllowed(bucket.bucketId, getUserId.value, Permissions.MANAGE );
+const getUserCanManageBucket = (bucket: Bucket) => {
+  return permissionStore.isBucketActionAllowed(bucket.bucketId, getUserId.value, Permissions.MANAGE );
 };
```

Comment:
We should ensure that we're also avoiding the get verb for component level actions here too. An alternative fn name could be `checkUserManageBucket` or similar. Would also be ideal for clarity/consistency sake that all our "actions" are written in oldschool function notation instead of arrow notation to help with at-a-glance visual clarity.